server/catgenome/src/test/java/com/epam/catgenome/manager/FeatureIndexManagerTest.java

```diff
@@ -540,6 +540,53 @@ public void testIndexUpdateOnProjectOperations() throws Exception {
 
     @Test
     @Transactional(propagation = Propagation.REQUIRES_NEW)
+    public void testAdditionalFilterWithVcfFileName() throws IOException {
+        Resource vcfResource1 = context.getResource(CLASSPATH_TEMPLATES_FELIS_CATUS_VCF);
```

Comment:
API for building feature index for project is obsolete and isn't used on UI anymore. Please, do not use calls to FeatureIndexManager methods with project ID (featureIndexManager.loadVcfFilterInfoForProject(project.getId())) for new tests.